tests/cpython-tests/Makefile

```diff
@@ -23,20 +24,29 @@ ext2fs: appdir
 	$(MYST) mkext2 appdir ext2fs
 
 clean:
-	rm -fr appdir ext2fs @test_101*
+	rm -fr appdir ext2fs @test_101* hostfs
 
 tests:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test -f tests.passed -q
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test -f tests.passed.1 -q
+	$(RUNTEST) ./exec.sh
 
-run-single:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test $(TC) -v
+#########################
+##	Dev targets
+#########################
+hostfs:
+	cp -r appdir/ hostfs/
```

Comment:
hostfs is not used anymore?

Reply:
Again a dev target. Hostfs is useful for debugging as you can change .py files and not have to rebuild the rootfs.

Reply:
I understand the conveniences. But if we little the code base in the main branch with dev-specific code, it will take much longer to make a release due to clean ups.

When a test is added, we should only work on it when there is a regression. That's why the "dev targets" are not that useful. 

Please keep the dev targets locally.

Reply:
There are still quite some failing test suites. The dev targets may aid in debugging those.

Reply:
The title of this PR is "cpython test suite fixes and cleanup". If we still keep the dev targets, that's not a clean up IMO.